pulp_container/app/tasks/sync_stages.py

```diff
@@ -68,11 +72,31 @@ async def run(self):
         """
         ContainerFirstStage.
         """
+
+        async def get_signature_source(self):
+            """
+            Find out where signatures come from: sigstore, extension API or not available at all.
+            """
+            if self.remote.sigstore:
+                return SIGNATURE_SOURCE.SIGSTORE
+
+            registry_v2_url = urljoin(self.remote.url, "v2")
+            extension_check_downloader = self.remote.get_noauth_downloader(url=registry_v2_url)
```

Comment:
To simplify our lives and not to create a downloader for HEAD request, I switched to using GET here, instead of HEAD.
HEAD is cheaper but this is called only once during sync.